ui/src/components/InOutSwitch/InOutSwitch.jsx

```diff
@@ -19,6 +19,7 @@ export default function InOutSwitch(props) {
     openValue,
     offValue,
     optionsOverlay,
+    invertBgColor,
```

Comment:
Just for explicitness:

```suggestion
    invertBgColor = false,
```